xDrip/Scenes/Entries/EditTraining/EditTrainingRouter.swift

```diff
@@ -25,4 +26,12 @@ final class EditTrainingRouter: EditTrainingRoutingLogic, EditTrainingDataPassin
     
     // MARK: Routing
     
+    func dismissSelf() {
+        switch dataStore?.mode {
+        case .edit:
+            viewController?.navigationController?.popViewController(animated: true)
+        default:
```

Comment:
Using `default` is usually a bad approach, because if you add other cases in the future, you can forget to add appropriate handling for it.